pages/advanced/cli-reference/common-operations.mdx

```diff
@@ -0,0 +1,253 @@
+import { Callout } from 'nextra/components'
+
+# Common Operations
+
+<Callout type='info' emoji='ℹ️'>
+  You must first load the sender's private key. When loading an owner, it will be set automatically.
+</Callout>
+
+## Safe
+
+### Create new Safe
+
+You can create a new Safe by running the following command:
```

Comment:
As this is a reference, all the params in the command should be explained in detail